### DISCUSSION TITLE
I lost the grid in my Visulaizer

### DISCUSSION THREAD

**Discussion by twohats60** [winder/Universal-G-Code-Sender#2373] [Help]
I am new to the CNC world. I have the Sainsmart 4040Pro and i have been messing around with UGS and I somehow lost the visualizer grid. I have the blank screen with the 4 icons in the top left corner but nothing else. I have eredownloaded as well with same issue. I know its something i did but can't figure what it was. I acn send files to UGS and carve the file. Any giudance would be apprecited,
Thanks,
Danny

**Comment by garyemiller** [comment]
MeToo
